MODULE.bazel

```diff
@@ -30,12 +30,201 @@ archive_override(
     urls = ["https://github.com/lowRISC/misc-linters/archive/refs/tags/20240823_01.tar.gz"],
 )
 
-# Includes:
-include("//third_party/rust:rust.MODULE.bazel")
+################### originally from //third_party/rust:rust.MODULE.bazel
 
```

Comment:
This big block of including individual module file contents was originally done on `master` as af68580259f.  You may want to double check just to make sure things are consistent.